frontend/components/ModalContainer.jsx

```diff
@@ -71,17 +86,29 @@ ModalContainer.propTypes = {
   })).isRequired,
   isLoading: PropTypes.bool.isRequired,
   onCloseFAQ: PropTypes.func.isRequired,
+  onConfirmAgeConsent: PropTypes.func.isRequired,
+  onRejectAgeConsent: PropTypes.func.isRequired,
   FAQModalVisible: PropTypes.bool.isRequired,
+  AgeConsentModalVisible: PropTypes.bool.isRequired,
```

Comment:
yeah, let's handle that in another issue. I think there are some other var naming issues where actions and selectors have colliding variable names. In this PR, for example, I had to rename the `showFAQModal` selector to `makeFAQModalVisible` because it had the same name as the `showFAQModal` action.